harness/src/test/java/org/jboss/pvt/harness/TestConfiguration.java

```diff
@@ -54,6 +76,33 @@ public Properties getAllConfiguration()
         return new String[0];
     }
 
+    @Override
+    public String getConfiguration(Class testClass, String key)
+    {
+        return getAllConfiguration().getProperty(testClass.getName() + "." + key);
+    }
+
```

Comment:
I would rather we stop adding new configuration API until PRODCI-24 is concluded